src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -58,41 +55,56 @@ function getActivitySummary({
   return baseSummary + suffix
 }
 
-// override the href prop when we dont want a clickable row
-const TransactionState = styled(OldTransactionState).attrs((props): { href?: string; isMeta?: boolean } => props)`
-  ${(props): string | false => !!props.isMeta && `pointer-events: none; cursor: none;`}
+// override the href, pending and success props
+// override mouse actions via CSS when we dont want a clickable row
+const TransactionState = styled(OldTransactionState).attrs(
+  (props): { href?: string; disableMouseActions?: boolean; pending?: boolean; success?: boolean } => props
```

Comment:
This is ugly, but I can't immediately suggest a non-ugly way to override required/optional prop
At a glance it would be `styled<infer actual comp from OldTransactionState,modify props>()`
Which is still ugly

Reply:
it's fine @Velenir haha, please let this one be. it'll be our secret

Reply:
👀